dom.bs

```diff
@@ -1074,12 +1083,14 @@ steps:
  <li><p>Let <var>capture</var> be the result of <a>flattening</a> <var>options</var>.
 
  <li><p>Let <var>once</var> and <var>passive</var> be false.
+ <li><p>Let <var>signal</var> be null.
```

Comment:
Newline before this. At least it seems we put each `<li>` on its own island from context.